### PR TITLE
chore(review): add automated review configuration

### PR DESCRIPTION
**Pull request by thxCode**
## Summary

- Add repository-specific Copilot review guidance for tenancy, migrations, generated clients, and security-sensitive code.
- Add the reusable automated code-review workflow.
- Add OpenCodeReview rules for Python services and the Helm chart, with configured exclusions.

## Verification

- Origin CI will run the repository test suite and required checks.
- Independently reviewed the frozen diff; no actionable findings.

### CODE REVIEW

File: .opencodereview/rule.json [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    },
+    {
+      "path": "gpustack/scheduler/**/*.py",
+      "merge_system_rule": true,
+      "rule": "Scheduling code. Flag candidate selection or scoring that violates requested resource constraints, worker availability, model compatibility, cluster/pool affinity, or multi-tenant visibility. Scheduling decisions must be deterministic for equivalent inputs and must not mutate shared state during evaluation. Flag resource arithmetic that can overcommit GPU, VRAM, RAM, or ports, and failure paths that strand reservations. Test changes with representative candidates and edge cases."
+    },
+    {
```
[the remark this answers is not in the document]
Comment (thxCode) [reply .opencodereview/rule.json:54]: Added a dedicated `gpustack/policies/**/*.py` rule covering worker filters, candidate selectors, and scorers with the relevant scheduling invariants.

File: .opencodereview/rule.json [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+      "path": "charts/gpustack-chart/**",
+      "merge_system_rule": true,
+      "rule": "GPUStack Helm chart. Flag templates that make configuration, resource, security, or lifecycle behavior diverge from charts/gpustack-chart/values.yaml and the server or worker runtime they configure. Ensure Secret values are never emitted into ConfigMaps, logs, or unsafe command arguments; generated Secrets must remain stable across upgrades where required. Flag invalid resource names, selectors, service ports, probes, volume mounts, RBAC, or server/worker topology. Chart.yaml dependency pins must remain compatible with their consumers: the gpustack-operator dependency pin tracks gpustack/__init__.py as enforced by tests/charts/test_operator_version_pin.py. Do not review vendored dependencies in charts/gpustack-chart/charts/; they are excluded."
+    }
+  ]
```
[the remark this answers is not in the document]
Comment (thxCode) [reply .opencodereview/rule.json:84]: Added explicit rules for `README.md` and `mkdocs.yml` covering translation-update notes and documentation navigation.

File: .github/workflows/code-review.yml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  # workflow only reads the diff and does not execute any code from the PR.
+  pull_request_target:
+    types: [opened, synchronize, reopened]
+  issue_comment:
+    types: [created]
+
+jobs:
+  review:
+    uses: gpustack/.github/.github/workflows/code-review.yml@main
+    with:
```
[the remark this answers is not in the document]
Comment (thxCode) [reply .github/workflows/code-review.yml:30]: Kept `@main` intentionally. This caller is byte-aligned with the supplied reference workflow, and the organization workflow explicitly centralizes policy updates for all adopting repositories. Pinning here would prevent that managed update path. Leaving this thread open for maintainer review.

File: .github/workflows/code-review.yml [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+  # available even for PRs from forks. This is safe because the reusable
+  # workflow only reads the diff and does not execute any code from the PR.
+  pull_request_target:
+    types: [opened, synchronize, reopened]
+  issue_comment:
+    types: [created]
+
+jobs:
+  review:
+    uses: gpustack/.github/.github/workflows/code-review.yml@main
```
[the remark this answers is not in the document]
Comment (thxCode) [reply .github/workflows/code-review.yml:29]: Kept the caller trigger-only. The reusable `gpustack/.github` workflow gates `issue_comment` events to pull requests, authorized human authors, and the supported re-review commands, so the condition remains centralized. Leaving this thread open for maintainer review.

**Review by gitlawr — APPROVED** [review]
LGTM
